libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="flex flex-row items-center">
   <span
-    class="align-text-top mr-1.5 favorite-count"
+    class="align-text-top mr-1.5"
+    data-test="favorite-count"
```

Comment:
We are already using `data-cy` in other locations. Maybe it's worth to name it the same. Then again I see it's also used in the UTs, so I let you judge :-)

Reply:
exactly, it used in the UT so I gave priority to `data-test` :)